src/options/generate/templates.py

```diff
@@ -0,0 +1,290 @@
+from langchain import PromptTemplate
```

Comment:
templates.py und prompt_system.py should be either unified or have unified names - user/ system